README_ZH.md

```diff
@@ -99,6 +99,13 @@ IoTDB提供了三种安装方法，您可以参考以下建议，选择最适合
 
 ## 从源码构建
 
+如果您使用Windows，请跳过此段。我们使用Thrift作为RPC模块来提供客户端-服务器间的通信和协议支持，因此在编译阶段我们需要使用Thrift 0.13.0
+（或更高）编译器生成对应的Java代码。Thrift只提供了Windows下的二进制编译器，Unix下需要通过源码自行编译。但我们预先编译了一个Thrift编译器，并将其上传到了GitHub
+，借助一个Maven插件，在编译时可以自动将其下载。该预编译的Thrift编译器在gcc8，Ubuntu, CentOS, MacOS下可以工作，但是在更低的gcc
+版本以及其他操作系统上尚未确认。如果您在编译时发现了Thrift编译器相关的问题，请升级您的gcc版本或者依照Thrift
+官方的指示自行编译编译器，并将编译器放置到目录`{project_root}\thrift\target\tools\thrift_0.12.0_0.13.0_linux.exe`。
+Thrift官方网址为：https://thrift.apache.org/
+
```

Comment:
More reasonable solution is :
add parameters in the maven command: 
1. If the user has installed thrift or has a binary thrift, then use `-Dthrift.download-url=http://apache.org/licenses/LICENSE-2.0.txt   -Dthrift.exec.absolute.path=<YOUR LOCAL THRIFT BINARY FILE>` 
2. If the user wants to download the binary thrift file from somewhere, then use `-Dthrift.download-url=<THE REMOTE URL FOR DOWNLOADING>   -Dthrift.exec.absolute.path=<THE DOWNLOADED BINARY FILE NAME>`

Reply:
Do you think we can add this description to readme ?

Reply:
I would keep both. The parameters are long and not that easy to remember, and adding them each time will not be handy. If you move the thrift binary properly, you will not need to add the parameters. Modifying the pom is another way, but it is dangerous for some one who is not familiar with pom.

Reply:
Keep both is fine.